server/app.js

```diff
@@ -19,6 +20,8 @@ const middlewares = [
 
 app.use(middlewares);
 
+dataBase.on('open', console.log.bind(console, 'mongo database is connecting'));
```

Comment:
It seems that ESLINT didn't work in your project ??  
console statment is not allowed so it should have thing like this 
```suggestion
// eslint-disable-next-line no-console
dataBase.on('open', console.log.bind(console, 'mongo database is connecting'));
```

Reply:
ok, I will solve it :+1: